src/OSPSuite.UI/Services/DataImporter.cs

```diff
@@ -180,11 +180,20 @@ DataImporterSettings dataImporterSettings
          dataSource.SetDataFormat(dataSourceFile.Format);
          dataSource.SetNamingConvention(configuration.NamingConventions);
          var sheets = new Cache<string, DataSheet>();
+         var missingSheets = new List<string>();
          foreach (var key in configuration.LoadedSheets)
          {
+            if (!dataSourceFile.DataSheets.Contains(key))
```

Comment:
Can we add some tests for this? Or is this too complicated at the moment?

Reply:
This is a good question. Adding a test here would mean testing the whole ImportFromConfiguration() function of the service. To test just this specific missing sheet we would then have to mock many things. I would suggest this becomes part of a new integration test, where excel files and conifgurations are being tested for import. 

Adding integration tests was in our ToDo list anyway, we could start with that as soon as the open PRs are finished.